src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -91,56 +124,71 @@ class CompanyStep extends React.Component {
         updateReimbursementAccountDraft(value);
     }
 
+    /**
+     * Clear the error associated to inputKey if found and store the inputKey new value in the state.
+     *
+     * @param {String} inputKey
+     * @param {String} value
+     */
+    clearErrorAndSetValue(inputKey, value) {
+        this.setValue({[inputKey]: value});
+        const errors = this.getErrors();
+        if (!errors[inputKey]) {
+            // No error found for this inputKey
+            return;
+        }
+
+        // Clear the existing error for this inputKey
+        const newErrors = {...errors};
+        delete newErrors[inputKey];
+        setBankAccountFormValidationErrors(newErrors);
+    }
+
     /**
      * @returns {Boolean}
      */
     validate() {
+        const errors = {};
         if (!this.state.password.trim()) {
-            showBankAccountFormValidationError(this.props.translate('common.passwordCannotBeBlank'));
-            return false;
+            errors.password = true;
```

Comment:
Now that we are checking if this has a value as a required field... can we remove this entirely?

Reply:
You are right there.. removing..